src/psij/web/lib.js

```diff
@@ -413,4 +413,15 @@ $(document).ready(function() {
             }
         });
     }, 500);
-});
\ No newline at end of file
+});
+
+//  For now, Used for Testing
```

Comment:
We should strive not to have unused or specific-issue debugging code in PRs and the main branch.

Reply:
That example code serves as documentation on how to test this call.  Otherwise, the tester is left with the painful process of figuring out how to call it.  do you still want me to remove it?

Reply:
If you intend to document how to invoke that call, then I would suggest clearly saying so. Additionally, if the code is only used for testing or documentation purposes, it can be put somewhere else, such as in a separate javascript file, a readme file, etc.